xla/service/gpu/transforms/command_buffer_scheduling.cc

```diff
@@ -97,6 +97,19 @@ static bool IsNoOp(const HloInstruction* hlo) {
 // done operation is not part of the same command buffer, we would change the
 // execution semantics and create additional synchronization point.
 
+// Returns true if the hlo instruction is async-start for dynamic-slice-fusion.
+static bool IsAsyncDynamicSliceFusion(const HloInstruction* hlo) {
+  HloInstruction* wrapped = hlo->async_wrapped_instruction();
+  absl::StatusOr<GpuBackendConfig> backend_config =
+      wrapped->backend_config<GpuBackendConfig>();
+  if (!backend_config.ok()) {
+    return false;
+  }
+  std::string name =
+      backend_config->fusion_backend_config().custom_fusion_config().name();
+  return name == "address_computation" || name == "dynamic_address_computation";
```

Comment:
Same comment as above re. `constexpr` (and `absl::string_view` on the line above, if it can be used).